python/flask-example/sample.py

```diff
@@ -46,4 +47,4 @@ def cmd_toolkit():
     return render_template('cmd.html', data=data)
 
 app.debug = True
-app.run(host='0.0.0.0', port=9000,)
+app.run(host='0.0.0.0', port=os.getenv('PORT', '9000'),)
```

Comment:
I don't think Flask allows the port to be specified as a string. You'd need to wrap this: `int(os.getenv('PORT', 9000))`

I'm not sure if bottle has the same limitation, but I suspect so.

Reply:
I'll add the int wrapper to the flask example. I have tested the `active-jobs-dashboard` locally and with ServiceCommander, and doesn't seem to enforce PORT being an int